run/filesystem/src/test/java/com/example/filesystem/ApplicationTests.java

```diff
@@ -131,8 +131,25 @@ public static void cleanup() throws IOException, InterruptedException {
         "--project=" + project);
 
     System.out.println("Deleting Cloud Run service: " + service);
-    Process p = delete.start();
-    p.waitFor(5, TimeUnit.MINUTES);
+    Process p1 = deleteService.start();
+    p1.waitFor(5, TimeUnit.MINUTES);
```

Comment:
If p1 and p2 are independent processes, we could optimize the wait time by moving `p1.waitFor()` between when we start p2 and `p2.waitFor()`. That would enable the container deletion start before we block on p1.